internal/capability/auditor.go

```diff
@@ -26,6 +29,79 @@ type CapAuditor struct {
 
 	// AllInstallModes will test all install modes supported by an operator
 	AllInstallModes bool
+
+	// ExtraCustomResources associates packages to a list of Custom Resources (in addition to ALMExamples)
+	// to be audited by the OperandInstall AuditPlan.
+	ExtraCustomResources map[string]interface{}
+}
+
+// ExtraCRDirectory scans the provided directory and populates the ExtraCustomResources field
+func (capAuditor *CapAuditor) ExtraCRDirectory(extraCRDirectory string) {
+	logger.Debugw("scaning for extra Custom Resources", "extra CR directory", extraCRDirectory)
+	extraCustomResources := map[string]interface{}{} // maps packages to a list of CR
+
+	extraCRDirectoryAbsolutePath, err := filepath.Abs(extraCRDirectory)
+	if err != nil {
+		logger.Errorf("Error getting absolute path of %s: %v", extraCRDirectory, err)
```

Comment:
This seems like something that should cause this to fail. You might want to change your signature to return an error.

In this case, you should return an error like:

```
return fmt.Errorf("could not get absolute path from %s: %v", extraCRDirectory, err)
```

Reply:
Addressed, as well as other comments regarding logging / returning error / continuing in 7bbad5d23beb1042de6f882ace95350ddad866d6